src/main/java/ladder/service/LadderGame.java

```diff
@@ -21,9 +24,35 @@ public static LadderGame getInstance() {
         return INSTANCE;
     }
 
-    public List<Line> createLadders(List<Member> members, int height) {
+    public List<Line> createLadders(List<Member> members, int height, CreateStrategy createStrategy) {
         return IntStream.range(0, height)
-                .mapToObj(num -> new Line(members.size(), RandomStrategy.getInstance()))
+                .mapToObj(num -> new Line(members.size(), createStrategy))
                 .collect(Collectors.toList());
     }
+
+    public LadderResult playLadders(List<Line> ladders, List<Member> members, List<Reword> rewords) {
+        if (members.size() != rewords.size()) {
+            throw new IllegalStateException("사다리 게임을 진행할 수 없습니다.");
+        }
+        List<Integer> point = getResultPoints(ladders, members.size());
+        return new LadderResult(getResultMap(members, rewords, point));
+    }
+
+    private static List<Integer> getResultPoints(List<Line> ladders, int memberCount) {
+        List<Integer> point = IntStream.range(0, memberCount)
+                .boxed()
+                .collect(Collectors.toList());
+        for (Line ladder : ladders) {
+            point = ladder.moveResult(point);
+        }
+        return point;
+    }
+
+    private static LinkedHashMap<Member, Reword> getResultMap(List<Member> members, List<Reword> rewords, List<Integer> point) {
+        LinkedHashMap<Member, Reword> map = new LinkedHashMap<>();
```

Comment:
```suggestion
        Map<Member, Reword> map = new LinkedHashMap<>();
```
구현체는 `LinkedHashMap` 쓰고 타입은 추상화해도 되지 않을까요?

Reply:
이게 사용하던 방식이랑 뭔가 달라서 이상하다 싶었는데 저런 실수를 했군요.,., 
수정했습니다!